[2023-01-03T00:11:31.462+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: uk_accidents_etl_pipeline.create_dashboard manual__2023-01-02T23:54:40.991487+00:00 [queued]>
[2023-01-03T00:11:31.507+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: uk_accidents_etl_pipeline.create_dashboard manual__2023-01-02T23:54:40.991487+00:00 [queued]>
[2023-01-03T00:11:31.508+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-03T00:11:31.509+0000] {taskinstance.py:1284} INFO - Starting attempt 3 of 4
[2023-01-03T00:11:31.510+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-03T00:11:31.567+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): create_dashboard> on 2023-01-02 23:54:40.991487+00:00
[2023-01-03T00:11:31.585+0000] {standard_task_runner.py:55} INFO - Started process 2525 to run task
[2023-01-03T00:11:31.603+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'uk_accidents_etl_pipeline', 'create_dashboard', 'manual__2023-01-02T23:54:40.991487+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/milestone_3.py', '--cfg-path', '/tmp/tmpoa_xm27d']
[2023-01-03T00:11:31.608+0000] {standard_task_runner.py:83} INFO - Job 28: Subtask create_dashboard
[2023-01-03T00:11:31.845+0000] {task_command.py:389} INFO - Running <TaskInstance: uk_accidents_etl_pipeline.create_dashboard manual__2023-01-02T23:54:40.991487+00:00 [running]> on host 82068c2abeb3
[2023-01-03T00:11:32.102+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_accidents_etl_pipeline
AIRFLOW_CTX_TASK_ID=create_dashboard
AIRFLOW_CTX_EXECUTION_DATE=2023-01-02T23:54:40.991487+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-02T23:54:40.991487+00:00
[2023-01-03T00:11:54.253+0000] {logging_mixin.py:137} INFO - Dash is running on http://0.0.0.0:8050/
[2023-01-03T00:11:54.252+0000] {dash.py:1968} INFO - Dash is running on http://0.0.0.0:8050/

[2023-01-03T00:11:54.284+0000] {logging_mixin.py:137} INFO -  * Serving Flask app '***'
[2023-01-03T00:11:54.288+0000] {logging_mixin.py:137} INFO -  * Debug mode: on
[2023-01-03T00:11:54.309+0000] {logging_mixin.py:137} WARNING - Address already in use
[2023-01-03T00:11:54.310+0000] {logging_mixin.py:137} WARNING - Port 8050 is in use by another program. Either identify and stop that program, or start the server with a different port.
[2023-01-03T00:11:54.524+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-01-03T00:11:54.617+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
